practice_independent_challenges/week4/colorado_lottery/lib/colorado_lottery.rb

```diff
@@ -0,0 +1,90 @@
+require 'Date'
+
+class ColoradoLottery
+    attr_reader :winners,
+                :current_contestants,
+                :registered_contestants,
```

Comment:
```suggestion
                :registered_contestants
```

I think if you delete this trailing comma on line 6 your tests will pass